web_readonly_bypass/static/src/js/readonly_bypass.js

```diff
@@ -138,4 +138,47 @@
         },
 
     });
+
+    instance.web.ProxyDataSet.include({
```

Comment:
You have to migrate the whole file to the new module-based JS API.

Reply:
Hello, do we have any update?

Reply:
JS has been ported here https://github.com/OCA/web/pull/743

Reply:
Can we close this one then?